src/Components/LoadingImg/LoadingImg.css

```diff
@@ -1,11 +1,11 @@
 #loadingImg {
-    height: 20vh;
-    width: 20vh;
-    animation: pulse 4s ease infinite;
+  height: 20vh;
+  width: 20vh;
+  animation: pulse 4s ease infinite;
 }
 
 @keyframes pulse {
-    0% { transform: scale(1); }
```

Comment:
Is there a setting on one of our text editors that causing this spacing issue?

Reply:
It might be a copy/paste thing, I think! @gaj23